backend/middleware/data_masking.py

```
@@ -45,24 +50,24 @@ def _initialize_masking_config(self) -> Dict[str, Any]:
-            
+
             # マスキングが必要かチェック
             if not self._should_mask_response(request, response):
                 return response
-            
+
             # レスポンスデータをマスキング
             masked_response = await self._mask_response_data(request, response)
-            
+
             return masked_response
```

Ruff (0.14.4)

Consider moving this statement to an `else` block

(TRY300)